addons/payment_stripe/models/payment_transaction.py

```diff
@@ -451,7 +451,7 @@ def _process_notification_data(self, notification_data):
                 self._set_error(_(
                     "The refund did not go through. Please log into your Stripe Dashboard to get "
                     "more information on that matter, and address any accounting discrepancies."
-                ))
+                ), extra_allowed_state='done')
```

Comment:
```suggestion
                message = _("quack quack")
                self._set_error(state_message=message, extra_allowed_state='done')
```
Either all the arguments can be kept in one line, or you make a line for each argument, like:
```
                self.method(
                    arg1,
                    arg2,
                    arg3,
                )
```